src/is-safari.ts

```diff
@@ -0,0 +1,11 @@
+function isSafari(ua?: string): boolean {
```

Comment:
Should we name it more specifically to `isDesktopSafari`? Or is the usage such that you would check `isDesktop` along with this 🤔 I recall the safari mobile v desktop stuff can be a bit...tricky

Reply:
I decided to name it this because the other desktop browsers are `is-chrome`, `is-firefox`, etc.
We also have `is-ios-safari` so I thought it'd be more clear to omit the "Desktop" part.

Reply:
🤔 maybe we add a little callout to the README explaining this naming convention?